### ISSUE TITLE
Failure to opening files with uppercase extension.

### ISSUE THREAD

**Issue by cetebuk** [java-decompiler/jd-gui#260]
OS: Win 7 Pro 64-bit
Java Env: JDK 8 Update 201
Tested versions: 1.4.3, 1.5.0, 1.6.3

If jar file has it's extension in uppercase (like: joo.JAR or FOO.JAR) file can not be opened for decompilation.

Steps to reproduce:
1. Opening test file "test.jar" (file opens without an issue)
2. Renaming the file to "test.JAR"
3. Menu -> Open file... selecting "test.JAR". 
4. Fails to Open.

**Comment by srinivasskc** [comment]
Is it the same issue for "App.Jar"? I am having a similar issue.

**Comment by srinivasskc** [comment]
> Is it the same issue for "App.Jar"? I am having a similar issue.

Yes, I tested with App.jar, It works fine. Does not work for App.Jar
